_hooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+
+# A git hook script that runs the same checks that are 
+# executed on Travis to verify your commits will not fail.
+# Bypass it with the --no-verify option to git-commit
+#
+# usage: make a symlink to this file,
+# e.g., rm -rf .git/hooks/ && ln -s -f ../_hooks .git/hooks
+
+# Run 
```

Comment:
Agreed, good eyes.  Removed that comment.

Fixed in https://github.com/m-lab/m-lab.github.io/pull/81/commits/3f6000223f6819c18a5133a60e77369a825dff00